src/App.tsx

```diff
@@ -15,20 +9,26 @@ import EligibilityRegistration from './components/registration/EligibilityRegist
 import Login from './components/login/Login'
 import Dashboard from './components/dashboard/Dashboard'
 import DownloadApp from './components/dashboard/DownloadApp'
-import { TopNav } from './components/widgets/TopNav'
 import GoogleAnalyticsPageTracker from './components/widgets/GoogleAnalyticsPageTracker'
 import Footer from './components/widgets/Footer'
 import DataRegulation from './components/static/DataRegulation'
 import PrivacyPolicy from './components/static/PrivacyPolicy'
 import Terms from './components/static/Terms'
 import ConsentInfo from './components/static/ConsentInfo'
+import { ThemeProvider, Typography } from '@material-ui/core'
```

Comment:
Great, could you move these import on top, so that all absolutes are first, then relative paths. Only to keep consistency in all files.